src/nginxconfig/scss/_panel.scss

```
@@ -38,7 +38,7 @@ THE SOFTWARE.
       display: flex;
       flex-direction: row;
       flex-wrap: wrap;
-      justify-content: space-between;
+      justify-content: start;
```

```suggestion
      justify-content: flex-start;
```
`start` has limited support, `flex-start` is preferred